### PR TITLE
Packaged 'perf-map-agent' and 'jmaps'.

### PR DESCRIPTION
**Pull request by fmap**

### CODE REVIEW

File: pkgs/default.nix [hunk 1]
```diff
@@ -169,6 +169,22 @@ in rec {
...
+  jmaps = stdenv.mkDerivation {
+    name = "jmaps";
+    src = fetchurl {
+      url = https://raw.githubusercontent.com/brendangregg/Misc/fc2856eb0d0d7fe7a3b3dfef8f873603f6a4a3eb/java/jmaps;
+      sha256 = "15bihx36qdjv25l1xdnkg3lrlmkjy9cmj6gsk4dybr141xfg9x4i";
+    };
+    unpackPhase = "true";
+    installPhase = ''
+      cp $src $out;
+      substituteInPlace $out --replace "/usr/lib/jvm/java-8-oracle" "${pkgs.openjdk}/lib/openjdk"
```
Comment (fmap) [inline pkgs/default.nix]: This is a bit hackish, is there a workaround?

Comment (fmap) [inline pkgs/default.nix]: Placement in patchPhase or something is fine, I'm just not sure about target paths with this style of `fetchurl`.

File: pkgs/default.nix [hunk 2]
```diff
@@ -171,6 +171,23 @@ in rec {
 
   jenkins = pkgs.callPackage ./jenkins {};
 
+  jmaps = stdenv.mkDerivation {
```
Comment (proger) [inline pkgs/default.nix]: could you put this one to its own file? i guess we should already start using `callPackage` for local dependencies..

Comment (fmap) [inline pkgs/default.nix]: Sure

**Pull request closed without merging**